frontend/src/scenes/session-recordings/player/inspector/playerInspectorLogic.ts

```diff
@@ -111,10 +111,8 @@ export const playerInspectorLogic = kea<playerInspectorLogicType>([
         ],
 
         syncScroll: [
-            true,
+            false,
```

Comment:
Gonna go for this but I'm unsure if this is what we want. I feel like having to click every time to turn on the scroll sync. 

Lets play with it but perhaps we want to have two "settings" - one for whether you overall want scrolling sync, and another that can be interrupted by things like expanding an item or manually scrolling away